### PR TITLE
Docker formatting improvements

### PR DESCRIPTION
**Pull request by jhinkey**

### CODE REVIEW

File: docs/dxp/7.x/en/installation-and-upgrades/installing-liferay/using-liferay-dxp-docker-images/dxp-container-lifecycle-and-api.md [hunk 1]
```diff
@@ -35,48 +35,93 @@ After you create a container in an environment, the container entry point execut
...
+    </tr>
+    <tr>
+        <th>File Location</th><th>Action</th><th>Use Cases</th>
+    </tr>
+    <tr>
+        <td><code>/usr/local/liferay/scripts/post-shutdown</code></td>
+        <td>Run scripts in alphabetical order</td>
+        <td><a href="./running-scripts-in-containers.md">Running scripts</a> after shutting down Tomcat</td>
+    </tr>
+</table>
```
Comment (jrhoun) [inline docs/dxp/7.x/en/installation-and-upgrades/installing-liferay/using-liferay-dxp-docker-images/dxp-container-lifecycle-and-api.md:124]: Hey @jhinkey i'm not sure we can get away with straight up using html to form the entirety of the table. Is there a better way to do this?

Comment (jhinkey) [reply docs/dxp/7.x/en/installation-and-upgrades/installing-liferay/using-liferay-dxp-docker-images/dxp-container-lifecycle-and-api.md:124]: The main thing I wanted to section off each phase API using rows that spanned all of the columns.

There might be a way to use `colspan=n` with Markdown tables. I'll check.

Comment (jrhoun) [reply docs/dxp/7.x/en/installation-and-upgrades/installing-liferay/using-liferay-dxp-docker-images/dxp-container-lifecycle-and-api.md:124]: ok - yeah i'm ok with some html here and there but to use it to create a full table makes me a bit uneasy.

Comment (jhinkey) [reply docs/dxp/7.x/en/installation-and-upgrades/installing-liferay/using-liferay-dxp-docker-images/dxp-container-lifecycle-and-api.md:124]: Column span doesn't seem to be supported in our Markdown. I'll send you a screenshot.

There's mention of GitHub flavored Markdown supporting it and mention of some Markdown parsers supporting it.

HTML is fully supported in Markdown. And the links work fine.

Comment (jhinkey) [reply docs/dxp/7.x/en/installation-and-upgrades/installing-liferay/using-liferay-dxp-docker-images/dxp-container-lifecycle-and-api.md:124]: The thing with tables is that all of their contents must be html.

That is, everything between the opening and closing `table` element tags

Comment (jhinkey) [reply docs/dxp/7.x/en/installation-and-upgrades/installing-liferay/using-liferay-dxp-docker-images/dxp-container-lifecycle-and-api.md]: OK. I reverted the table to be in Markdown. +1

**Comment by jrhoun** [comment]
Merged. Thanks! Removed the outline sections here: https://github.com/liferay/liferay-learn/commit/1c46ac8fd96e9999854962c82f101631e15c3480

**Pull request closed without merging**